addons/hr/tests/test_self_user_access.py

```diff
@@ -209,12 +209,6 @@ def testWriteOtherUserPreferencesEmployee(self):
             with self.assertRaises(AccessError):
                 self.hubert.with_user(self.richard).write(v)
 
-    def testWriteSelfPhoneEmployee(self):
-        # phone is a related from res.partner (from base) but added in SELF_READABLE_FIELDS
-        self.env['ir.config_parameter'].set_param('hr.hr_employee_self_edit', False)
```

Comment:
Shouldn't we update ``testWriteOtherUserPreferencesEmployee`` hereabove to show it is now editable ?

Reply:
@tde-banana-odoo Shouldn't it be better to update `testWriteSelfUserPreferencesEmployee`, as it tests whether the user can update `their` non-employee fields without raising an error? (If I misunderstood, please correct me.)

Reply:
Yes indeed ^^